plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/IRouteChooser.java

```diff
@@ -20,6 +20,17 @@
 public interface IRouteChooser {
 
 	public IRoute chooseRoute(List<IRoute> routes);
+
+	/**
+	 * Returns user's decision to remember the chosen route.
+	 * This method should be called after chooseRoute(List<IRoute).
+	 * 
+	 * @return
+	 */
+	default public boolean isRememberChoice() {
```

Comment:
I have no issues correcting the other method signatures either.  It would be obvious from the diff what is happening and that no code is being changed.